docs/docs/guides/airy-core-and-facebook.md

```diff
@@ -0,0 +1,44 @@
+---
+title: How to connect the Airy Core Platform to Facebook
+sidebar_label: Connecting Facebook
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::tip What you will learn
+- How to connect Airy Core Platform to Facebook
+:::
+
+In the Airy Core Platform we consider Facebook to be a [Source](glossary.md#source),
+meaning we can connect it to Airy using a [Channel](glossary.md#channel).
+
+Facebook provides a mechanism that sends messages in a structured format to your local Airy instance.
```

Comment:
this means the webhook? If so, I think we can call it like that. It's a technical audience after all